src/super_gradients/training/utils/callbacks/callbacks.py

```diff
@@ -146,16 +137,12 @@ class DeciLabUploadCallback(PhaseCallback):
 
     def __init__(self, model_meta_data, optimization_request_form, ckpt_name="ckpt_best.pth", **kwargs):
         super().__init__(phase=Phase.POST_TRAINING)
-        if _imported_deci_lab_failure is not None:
-            raise _imported_deci_lab_failure
```

Comment:
same here. you need to add an if-raise statement to `DeciClient.upload_model()` and `is_model_benchmarking()`

Reply:
See previous comment